pytest_django/fixtures.py

```diff
@@ -6,6 +6,10 @@
 
 import pytest
 
+from contextlib import contextmanager
+from django.db import connection
+from django.test.utils import CaptureQueriesContext
```

Comment:
These imports might cause the test failures.
Try importing it in the new fixture.